modules/data_warehouse/main.tf

```diff
@@ -460,6 +461,18 @@ resource "google_project_iam_member" "pubsub" {
   member = "serviceAccount:${data.google_storage_project_service_account.gcs_account.email_address}"
 }
 
+# # Sleep for a few minutes to let Eventarc sync up
+resource "time_sleep" "wait_for_eventarc" {
+  create_duration = "180s"
+  
```

Comment:
linter seems to be complaining
./modules/data_warehouse/main.tf:467:  
Error: Trailing whitespace found in the lines above.


```suggestion

```

Reply:
fixed